CMakeLists.txt

```diff
@@ -22,6 +22,32 @@ set(GMOCK_VER "1.7.0" CACHE STRING "Google Mock framework version to be used")
 
 set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake/modules)
 
+###############
+# Conan support
+###############
+
+find_program (CONAN_BIN conan)
+if(CONAN_BIN)
+    message (STATUS "Found conan C++ package manager: ${CONAN_BIN}")
+    find_file (CONANFILE NAMES "conanfile.txt" HINTS "${CMAKE_SOURCE_DIR}")
+    if (CONANFILE)
+        message (STATUS "Found ${CONANFILE}")
+        if (NOT EXISTS "${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
+            message(STATUS "Trying to execute 'conan install'")
+            get_filename_component(CONANFILE_DIR ${CONANFILE} DIRECTORY)
+            execute_process(COMMAND conan install ${CONANFILE_DIR} --build=missing)
```

Comment:
This is something that shouldn't be done from the build system. It should be considered a per-requisite to executing the build system, nothing more.

Executing `conan install` should not be considered a significant burden on the user. That should be added to the README though.